src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -0,0 +1,53 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.CartDto;
+import com.kodilla.ecommercee.domain.dto.CartItemDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping(value = "/carts")
+public class CartController {
+    
+    @PostMapping
+    public ResponseEntity<Void> createCart() {
+        List<CartItemDto> cart = new ArrayList<>();
+        //post to repository//
+        return ResponseEntity.ok().build();
+    }
```

Comment:
We should know for which user we are creating this cart. My syggestion:
@PostMapping(value = "/createCart/{userId}")
 public ResponseEntity<Void> createCart(@PathVariable Long userId) {
        List<CartItemDto> cart = new ArrayList<>();
        //post to repository//
        return ResponseEntity.ok().build();
    }

Reply:
OK.